keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -417,12 +420,9 @@ expression references to the parameters via the CEL `params` variable, e.g.
 validate. This also guides type-checking, see the "Informational type checking"
 section for details.
 
-`denyReason` assigns a status code the result of this policy if it evaluates to
-deny.
-
 The `spec.validations` fields contain CEL expressions. If an expression
-evaluates to false, the validation check is enforced (the below `PolicyBinding`
-configures the enforcement).
+evaluates to false, the validation check is enforced according to the
+`enforcement` field.
```

Comment:
Did you mean to both to say this and also remove the field from the examples?

Reply:
Sorry, this might be confusing. This references the policy definition `enforcement` field, I've added back the `mode: Enabled` to the policy bindings. It should read right now.